src/redux/user/user.actions.js

```diff
@@ -1,31 +1,31 @@
 import {
   LOGIN_USER,
   SET_USER,
-  SET_ERROR,
   LOGOUT_USER,
-  USER_LOADING
+  SET_USER_ERROR,
+  SET_USER_LOADING
 } from './user.types';
 
 const setUser = (user) => ({
   type: SET_USER,
   payload: user
 });
 
-const loginUser = (user) => ({
+const loginUser = (payload) => ({
   type: LOGIN_USER,
-  payload: user
+  payload
 });
 
-const setError = (error) => ({
-  type: SET_ERROR,
+const setUserError = (error) => ({
+  type: SET_USER_ERROR,
   payload: error
 });
 
 const logoutUser = () => ({
   type: LOGOUT_USER
 });
 
-const userLoading = () => ({
-  type: USER_LOADING
+const setUserLoading = () => ({
```

Comment:
I think `setUserLoading` should take a boolean argument to show or not show loading